### ISSUE TITLE
Suggestion: add ipconfig /flushdns

### ISSUE THREAD

**Issue by gitmuti** [bleachbit/bleachbit#2015]
Suggestion: 

Enable the optional execution (using cmd internally) of the following command 
**ipconfig /flushdns**

Justification:
See e.g. https://monsterhost.com/why-flush-dns-how-to-do-it/

**Comment by lorimode** [comment]
Yes, I was going to open a suggestion also for that, it would really be a plus for bleachbit 👍

**Comment by az0** [comment]
I am closing as duplicate to https://github.com/bleachbit/bleachbit/issues/1698 . You may want to visit that page, click the "subscribe" button to get updates. You may also want to add a reaction (like thumbs up) to the top post.

**Comment by az0** [comment]
[BleachBit version 6.0.1 beta](https://www.bleachbit.org/news/bleachbit-601-beta) was released with the DNS cleaner. When you have a chance, try it out.
